staging/src/k8s.io/cri-api/README.md

```diff
@@ -9,6 +9,189 @@ The repository [kubernetes/cri-api](https://github.com/kubernetes/cri-api) is a
 Please do NOT file issues or submit PRs against the [kubernetes/cri-api](https://github.com/kubernetes/cri-api)
 repository as it is readonly, all development is done in [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes).
 
+CRI API is defined in [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes)
+repository and is ONLY intended to be used for kubelet to container runtime interactions.
+It is NOT a common purpose container runtime API for general use and intended
+to be Kubernetes-centric. We try to avoid it, but there may be logic in container
+runtime optimized for the order or specific parameters of call kubelet makes.
```

Comment:
- Markdown has native support for strong emphasis.
- `crictl` is a thing

```suggestion
The CRI API is defined in [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes)
repository and is **only** intended to be used for kubelet to container runtime 
interactions, or for node-level troubleshooting using a tool such as `crictl`.
It is **not** a common purpose container runtime API for general use, and is intended
to be Kubernetes-centric. We try to avoid it, but there may be logic within a container
runtime that optimizes for the order or specific parameters of call(s) that the kubelet
makes.
```